featuretools/tests/primitive_tests/test_groupby_transform_primitives.py

```diff
@@ -352,6 +355,32 @@ def get_function(self):
         assert ((pd.isnull(x) and pd.isnull(y)) or x == y)
 
 
+def test_groupby_multi_output_stacking(es):
+    def test_time(x):
+        return x
```

Comment:
this is a miss on codecov because we don't actually calculate the feature values

if we do something like this:
```python
TestTime = make_trans_primitive(
    function=lambda x: x,
    name="test_time",
    input_types=[Datetime],
``` 

codecov won't complain and the test will still run

Reply:
the issue is that the name of the primitive then becomes "lambda" instead of test_time in the name of the primitive
<img width="764" alt="Screen Shot 2019-08-08 at 4 51 43 PM" src="https://user-images.githubusercontent.com/14035415/62737003-d2244400-b9fc-11e9-8a01-31aeecb89033.png">

Reply:
I believe if you add `name="test_time` as a parameter it won't use the lambda to come up with the feature name

Reply:
oh missed that part, thanks!